File: src/main/java/org/example/Java/Act3.java
package org.example.Java;

import java.util.Scanner;

public class Act3 {
    Scanner scanner = new Scanner(System.in);
    public String Area() {
        System.out.println("De cuanto es tu radio ?");
        double radio = Double.parseDouble(scanner.nextLine());
        double area = Math.PI * Math.pow(radio, 2);
        return "El area del circulo sería :  " + area;
    }
}

File: src/main/java/org/example/Kotlin/Act3.kt
package org.example.Kotlin

import kotlin.math.pow

class Act3 {
    fun Area(): String {
        println("De cuanto es tu radio ?")
        val radio: Double = readLine()!!.toDouble()
        val area = Math.PI * radio.pow(2.0)
        return "El area del circulo sería :  $area"
    }
}